server/routes/service.test.js

```diff
@@ -92,137 +93,146 @@ describe('Serivice layer unit test', () => {
 
     const result = await service.signUpDbQuery('correct_name', 'correct_pw');
 
-    expect(result).to.equal('500');
+    expect(result).to.equal('401');
   });
 
-  it('Check an error returned from user database and returns 500 as a function', async () => {
-    sinon.stub(db, 'checkUp').returns(Error());
+  it('Should return access token', async () => {
+    sinon.stub(db, 'createUser').returns({ user_id: 9999, user_name: 'nonexisting_name' });
+    sinon.stub(jwt, 'sign').returns('accessToken');
 
-    const result = await service.signUpDbQuery('correct_name', 'correct_pw');
+    const result = await service.signUpDbQuery('nonexisting_name', 'nonexisting_pw');
 
-    expect(result).to.equal('500');
+    expect(result).to.equal('accessToken');
   });
 
-  it('Check new user record created', async () => {
-    sinon.stub(db, 'createUser').returns({ user_id: 1, user_name: 'correct_name' });
-    sinon.stub(jwt, 'sign').returns('accessToken');
+  it('Should return 500 and fail creating new user for null user name', async () => {
+    sinon.stub(db, 'createUser').returns({ user_id: 9999, user_name: null });
 
-    const result = await service.signUpDbQuery('correct_name', 'correct_pw');
+    const result = await service.signUpDbQuery('nonexisting_name', 'nonexisting_pw');
 
-    expect(result).to.equal('accessToken');
+    expect(result).to.equal('500');
   });
 
-  it('Check new user record created but user id not provided and returns 500', async () => {
-    sinon.stub(db, 'createUser').returns({ user_id: null, user_name: 'correct_name' });
+  it('Should return 500 and fail creating new user for null user id', async () => {
+    sinon.stub(db, 'createUser').returns({ user_id: null, user_name: 'nonexisting_name' });
 
-    const result = await service.signUpDbQuery('correct_name', 'correct_pw');
+    const result = await service.signUpDbQuery('nonexisting_name', 'nonexisting_pw');
 
     expect(result).to.equal('500');
   });
 
   // Test for function tokenCheck
-  it('Check cookie contains token information and returns if it is null', async () => {
+  it('Should return 401 for null cookie', async () => {
     const result = service.tokenCheck(null);
 
     expect(result).to.equal('401');
   });
 
-  it('Check an inputted sign in information is invalid and emmits error and returns 403', async () => {
-    sinon.stub(jwt, 'verify').returns('403');
+  it('Should return 403 for invalid user input', async () => {
+    sinon.stub(jwt, 'verify').withArgs('incorrect_information').returns('403');
 
     const result = service.tokenCheck('incorrect_information');
 
     expect(result).to.equal('403');
   });
 
-  it('Check an inputted sign in information is valid and returns user information', async () => {
-    sinon.stub(jwt, 'verify').returns('user_information');
+  it('Should return user information for valid sign in user input', async () => {
+    sinon.stub(jwt, 'verify').withArgs('correct_information').returns('user_information');
```

Comment:
I believe the user information is not in a strong, but most likely in an object 🤔